### DISCUSSION TITLE
KeePass

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#11] [Requests] [answered]
KeePass website: https://keepass.info/

KeePass privacy policy: https://keepass.info/help/base/terms.html

Logo: https://upload.wikimedia.org/wikipedia/commons/0/04/KeePass_icon.svg

Description: "KeePass Password Safe is a free and open-source password manager primarily for Windows. It officially supports macOS and Linux operating systems through the use of Mono. Additionally, there are several unofficial ports for Windows Phone, Android, iOS, and BlackBerry devices. KeePass stores usernames, passwords, and other fields, including free-form notes and file attachments, in an encrypted file. This file can be protected by any combination of a master password, a key file, and the current Windows account details. By default, the KeePass database is stored on a local file system (as opposed to cloud storage).

KeePass supports a number of plugins. It has a password generator and synchronization function, supports two-factor authentication, and has a Secure Desktop mode. It can use a two-channel auto-type obfuscation feature to offer additional protection against keyloggers. KeePass can import from over 30 other most commonly used password managers.

A 2017 Consumer Reports article described KeePass as one of the four most widely used password managers (alongside 1Password, Dashlane and LastPass), being "popular among tech enthusiasts" and offering the same level of security as non-free competitors.

A 2019 Independent Security Evaluators study described KeePass as well as other widely used password managers as being unable to control Windows 10's tendency to leave passwords in cleartext in RAM after they are displayed using Windows controlled GUI. In addition, several github projects (KeeFarce, KeeThief, Lazanga) specifically attack a running KeePass to steal all data; when the host is compromised. KeePass cannot prevent password theft and neither KeePass nor any other password manager can magically run securely in a spyware-infected, insecure environment."

**Comment by mayrinck** [comment, marked as answer]
This might be a tough one... Mostly because it deal with passwords and such. Still, I will try to add it in the future. Thank you for submitting!
